spec/stack.js

```diff
@@ -19,4 +18,59 @@ describe('Stack', () => {
         .to.alter(() => myStack.length(), { from: 0, to: 1 })
     })
   })
+
+  context('pop()', () => {
+    it('returns and removes element from top of the stack.', () => {
```

Comment:
You would also want to include a test to ensure that the *correct value* was popped.